.github/workflows/ci_test.yml

```diff
@@ -14,7 +14,33 @@ env:
   TOXARGS: '-v'
 
 jobs:
+
+  check_commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+# Found this solution at 
+# https://monadical.com/posts/filters-github-actions.html#Case-2-Pull-request
+      - name: check if message contain keywords doc skip-tests
+        id: check_commit
+        run: |
+          message=$(git log -1 --pretty=format:'%s')
+          re="\[(doc|skip-tests).*\]"
+          if [[ $message =~ $re ]]; then
+            echo "::set-output name=match::true"
```

Comment:
`set-output` is deprecated. Consider https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Great catch! Thanks a lot!